app/models/feedbook/employee.rb

```diff
@@ -1,11 +1,10 @@
 module Feedbook
   class Employee < ApplicationRecord
-    has_ancestry
```

Comment:
@diegosaul402 we need to this line because we use ancestry gem, this gem allows us to change our model in a tree with descendants, could you recover this linea again?

Reply:
Oh ok, I'll do it